internal/controllers/routerinterface/reconcile.go

```diff
@@ -99,21 +95,20 @@ func (r *orcRouterInterfaceReconciler) Reconcile(ctx context.Context, req ctrl.R
 		routerInterfacePorts = append(routerInterfacePorts, *port)
 	}
 
-	errs := make([]error, len(routerInterfaces))
+	// TODO: refactor this loop so reconcileNormal and reconcileDelete both use and return progressStatus
```

Comment:
Do you want to create an issue for this?

Reply:
Can do